src/hooks/initializeSDK.ts

```diff
@@ -12,7 +12,7 @@ import {set} from "../lib/SDK";
  * dumped to the console. So to handle that we subscribe to that event and use the internal oclif error handling to print the error in the same
  * way that all other this.error calls happen.
  */
-process.on("unhandledRejection", (error: Error) => handle(new CLIError(error)));
+process.on("unhandledRejection", (reason): void => handle(new CLIError(reason as Error)));
```

Comment:
This is weird. What was up with this?

Reply:
Node 12 changed the types for this event for some reason. The types are now

```
on(event: "unhandledRejection", listener: UnhandledRejectionListener): this;

type UnhandledRejectionListener = (reason: {} | null | undefined, promise: Promise<any>) => void;
```

So now they also provide you with the promise that failed. I poked at it some more after your comment and I think I made it better now.